executor/analyze.go

```diff
@@ -382,6 +390,12 @@ func (e *AnalyzeExec) handleResultsErrorWithConcurrency(ctx context.Context, sta
 		}
 		err = errors.New(strings.Join(errMsg, ","))
 	}
+	for tableID := range tableIDs {
+		// Dump stats to historical storage.
+		if err := recordHistoricalStats(e.ctx, tableID); err != nil {
+			logutil.BgLogger().Error("record historical stats failed", zap.Error(err))
+		}
+	}
```

Comment:
Though we save stats concurrently by several workers, the logic of recording historical stats is sequential. Will we improve it in the future?

Reply:
dumping historical stats is worked by multiple worker in background, thus it's not sequential